_config/config.yml

```diff
@@ -2,9 +2,12 @@ SilverStripe\Assets\File:
   allowed_extensions:
     - crt
     - key
+    - pem
```

Comment:
Adding `.pem` to our allowed list of extensions

Reply:
Good shout, no longer need `.key` but still need `.crt` for the _encryption_ cert, anything _signing_ related uses `.pem`